sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -22,6 +22,9 @@ enablePythonPerformanceTest()
 String pythonVersionSuffix = project.ext.pythonVersion
         ? "-py${project.ext.pythonVersion.replace('.', '')}"
         : ''
+// Python versions to run secondary integration tests.
+// Should set to each minimum supported minor version.
+def runItDfSecondaryVersions = ['3.7']
```

Comment:
We already have a couple of places where we configure python versions:
https://github.com/apache/beam/blob/master/sdks/python/test-suites/gradle.properties
https://github.com/apache/beam/blob/master/.test-infra/jenkins/PythonTestProperties.groovy
I'd prefer not to add yet another place. 

I think the fix should be separating IO tests into a separate suite. @MarcoRob and team are looking into migrating to GH Actions and if I understand the plan, they are sharding the tests into separate suites as part of that effort. 
IO GCP would become a separate suite, is that correct, @MarcoRob ? 
@Abacn if we need to break it down further now may be a good time to provide feedback.

Reply:
Ok, Since splitting test suites during GA migration is ongoing, better defer this change to avoid conflicts. Will come back to see how things going once the migration done.

Reply:
Yes, that is correct @tvalentyn, we are splitting the jobs, so it is a good idea to wait until the migration is complete and validate the test and the change (if needed) that it would require on this. Tks